test/test_ops.py

```diff
@@ -2195,7 +2195,15 @@ def test_fake_crossref_backward_no_amp(self, device, dtype, op):
     @skipOps('TestFakeTensor', 'test_fake_crossref_backward_amp', fake_backward_xfails | fake_autocast_backward_xfails)
     def test_fake_crossref_backward_amp(self, device, dtype, op):
         self._test_fake_crossref_helper(device, dtype, op, torch.cuda.amp.autocast)
-
+    @ops([op for op in op_db if op.is_factory_method])
```

Comment:
Leave an empty line between tests

Reply:
Shouldn't 'op.is_factory_method` be `op.is_factory_function`?